straxen/url_config.py

```diff
@@ -0,0 +1,201 @@
+
+import json
+import strax
+import fsspec
+import straxen
+from numpy import isin
+import inspect
+from urllib.parse import urlparse, parse_qs
+from ast import literal_eval
+from functools import lru_cache
+
+from strax.config import OMITTED
+
+export, __all__ = strax.exporter()
+
+
+def parse_val(val):
+    try:
+        val = literal_eval(val)
+    except:
```

Comment:
```suggestion
    except Exception:
```
To prevent excepting baseExceptions:
```
[pep8] reported by reviewdog 🐶
B001 Do not use bare `except:`, it also catches unexpected events like memory errors, interrupts, system exit, and so on.  Prefer `except Exception:`.  If you're sure what you're doing, be explicit and write `except BaseException:`.
```